docker-compose.yml

```diff
@@ -8,22 +45,29 @@ services:
       - mongo
       - rabbit
       - ipfs
-      - bitcoin-regtest
+      - bitcoind-1
     environment:
-      - BITCOIN_URL=bitcoin-regtest
-      - RABBITMQ_URL=amqp://poet-rabbit
-      - MONGODB_URL=mongodb://poet-mongo:27017/poet
-      - IPFS_URL=http://poet-ipfs:5001
+      - POET_SERVICE=node
+      - BITCOIN_URL=bitcoind-1
+      - RABBITMQ_URL=amqp://rabbit
+      - MONGODB_HOST=mongo
+      - IPFS_URL=http://ipfs:5001
       - INTEGRATION_TEST_MONGODB_URL=mongodb://poet-mongo:27017/poet
```

Comment:
Should be `mongodb://mongo:27017/poet` ?

Reply:
It's not actually being used anywhere - I will remove them.